### ISSUE TITLE
themeConfig not changing form

### ISSUE THREAD

**Issue by jpeacock** [verbb/formie#986]
### Describe the bug

Using the most basic example on a form, `themeConfig` seems to not affect the form at all. The form renders, any `themeConfig` values do not change classes, id's, etc on any element inside of the form. 

```
{{ craft.formie.renderForm('contactForm', {
    themeConfig: {
        form: {
            attributes: {
                id: 'my-form',
            },
        },
    },
}) }}
```

### Steps to reproduce

1. Create a form in the Craft control panel.
2. Reference it in a template via its handle.
3. Use `themeConfig` to apply a different id to the form. 

### Form settings

- Multi-page form: No
- Submission Method: Ajax and Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.1.3

### Plugin version

2.0.0-beta.16

### Multi-site?

No

**Comment by jpeacock** [comment]
Hah, just saw the beta was updated to the 2.0.0 release 2 days ago. Checked and it works now with inline config settings. Settings in formie.php do not yet.

**Comment by engram-design** [comment]
The config file should work (it does for me) can you share your config file just in case that might be formatted incorrectly?

**Comment by jpeacock** [comment]
So, I noticed that if I don't have any sort of themeConfig in the twig template, but I have a config in the config.php file, it doesn't use any of the themeConfig. 
![Screen Shot 2022-07-13 at 9 50 50 PM](https://user-images.githubusercontent.com/223784/178887777-e2aa3b60-e7c5-41ab-a1eb-3e8e8ac6143b.png)

But, if I have a themeConfig of some sort in the twig template, and also have a global one in config.php, it then applies both. 
![Screen Shot 2022-07-13 at 9 52 46 PM](https://user-images.githubusercontent.com/223784/178887992-5684dd46-7bfd-4dad-b796-dd2be4e6b05d.png)

**Comment by engram-design** [comment]
Ah, you're totally right! Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jpeacock** [comment]
awesome! thanks!

**Comment by engram-design** [comment]
Fixed in [2.0.2](https://github.com/verbb/formie/releases/tag/2.0.2)

**Comment by engram-design** [comment]
Fixed in [2.0.2](https://github.com/verbb/formie/releases/tag/2.0.2)
